uisdk/src/main/java/com/karhoo/uisdk/screen/booking/booking/tripallocation/TripAllocationView.kt

```diff
@@ -66,6 +67,8 @@ class TripAllocationView @JvmOverloads constructor(
         cancelButton.isEnabled = true
         cancelButton.setListener { cancelTrip() }
         presenter?.waitForAllocation(trip)
+        handler.postDelayed({ presenter?.handleAllocationDelay() }, ALLOCATION_ALERT_DELAY)
```

Comment:
I wonder what happens when we are in the Guest mode and the user is navigated to the WebTracker but this timer still running, do you think we should schedule this thread only if it is not Guest mode?

Reply:
Done